pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/GjenlevendeInfoFoer1970.kt

```diff
@@ -0,0 +1,132 @@
+package no.nav.pensjon.brev.maler.adhoc
```

Comment:
Skal disse brevene kun være på bokmål?

Reply:
Tekstene er ikke ferdige enda, og ikke ferdig oversatt. Disse versjonene er kun for å teste bestillings-løpet.